IdentityCore/src/parameters/MSIDRequestParameters.h

```diff
@@ -65,6 +65,11 @@
 #if !EXCLUDE_FROM_MSALCPP
 @property (nonatomic) MSIDCurrentRequestTelemetry *currentRequestTelemetry;
 #endif
+
+#pragma mark Nested auth protocol properties
+@property (nonatomic) NSString *nestedClientId;
```

Comment:
See other comments, let's use consistent name for this property. Should be nestedBrokerClientId, nestedBrokerRedirectUri

Reply:
Updated